src/utils/dom-to-image.js

```diff
@@ -470,7 +476,16 @@ function newUtil() {
       image.onload = () => {
         resolve(image);
       };
-      image.onerror = reject;
+      image.onerror = (err) => {
+        const message = `[kepler.gl] Failed to create image from data uri.
```

Comment:
define a constant for this one in constant file here: https://github.com/keplergl/kepler.gl/blob/master/src/constants/user-feedbacks.js

Reply:
done